modules/core/src/utils/tesselator.js

```diff
@@ -101,7 +100,10 @@ export default class Tesselator {
   }
 
   getGeometryFromBuffer(geometryBuffer) {
```

Comment:
Nit: If `geometryBuffer` is not always a `Buffer` object, consider renaming this method?

Reply:
We call both ArrayBuffer and luma Buffer `Buffer` in this code base.

I will do some refactor in a follow up PR after the patch.